<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VigenereCipherMachine</class>
 <widget class="QMainWindow" name="VigenereCipherMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>678</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>678</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>678</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vigenere Cipher Machine</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="KeyPhraseBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter your key phrase here....</string>
    </property>
   </widget>
   <widget class="QLabel" name="CiphertextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>305</y>
      <width>781</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ciphertext should appear here....</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="PlaintextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>781</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter your text here to encrypt....</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UploadTxtFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Upload .txt File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DownloadTxtFileButton">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>590</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Download .txt ciphertext file</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
